### DISCUSSION TITLE
Set a field to null

### DISCUSSION THREAD

**Discussion by jghauser** [kkharji/sqlite.lua#151] [Q&A] [answered]
Is there a way of using `update` to set a field to `null`? When I set the value to `nil` then it's not passed on to sqlite at all (because lua considers that key-value pair as inexistent). When i set it to `null`, I just get the string "null" (or an error if the type isn't `string`). Thanks!

**Comment by kkharji** [comment, marked as answer]
Oh that's a tricky use case I did not consider, or might've but overlooked the usage of null as removing something 🤔. I don't think it's possible with high level api. You would need to use sqlite.db methods, like `sqlite.db:eval` or `sqlite.db:execute`. 

If you can update the whole row, then it might work.

**Reply by jghauser** [reply]
Thanks for the quick reply! I'll go with the db methods then :)

**Comment by jghauser** [comment]
@kkharji, I just wanted to say thanks for all the support. I've now published [papis.nvim](https://github.com/jghauser/papis.nvim/) which wouldn't have been possible without all your effort!
